src/Sentry/ITransactionData.cs

```diff
@@ -1,8 +1,59 @@
+using Sentry.Protocol;
+
 namespace Sentry;
 
 /// <summary>
 /// Immutable data belonging to a transaction.
 /// </summary>
 public interface ITransactionData : ISpanData, ITransactionContext, IEventLike
 {
+    /// <summary>
+    /// The measurements that have been set on the transaction.
+    /// </summary>
+    IReadOnlyDictionary<string, Measurement> Measurements { get; }
+
+    /// <summary>
+    /// Sets a measurement on the transaction.
+    /// </summary>
+    /// <param name="name">The name of the measurement.</param>
+    /// <param name="measurement">The measurement.</param>
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Why do we want this not to be visible in the IDE?

Reply:
I think that was copied across verbatim. I'll get rid of it.